src/script/stores/mlStore.ts

```diff
@@ -129,19 +156,21 @@ const initialMLSettings: MlSettings = {
     Filters.ACC,
     Filters.ZCR,
     Filters.RMS,
+    Filters.GRAD,
   ]),
+  dataSource: DataSource.ACCELEROMETER,
 };
 
 // Store with ML-Algorithm settings
-export const settings = persistantWritable<MlSettings>('MLSettings', initialMLSettings);
+export const settings = writable<MlSettings>(initialMLSettings);
```

Comment:
Don't persist mlSettings to localStorage. These settings cannot currently be set by the user and storing these data in localStorage is making it tricky to update them independently of other keyed data in localStorage. See `persistVersion` in storeUtil.ts.

Reply:
Sensor selection, which is user controlled, has been pulled out into `persistentSettings` which is persisted to `localStorage`.